docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+services:
+    db:
+        image: mysql:8
+        container_name: mapbot_db
+        environment:
+            - MYSQL_ROOT_PASSWORD=root
+            - MYSQL_DATABASE=mapbot
+        volumes:
+            - ./db-init/:/docker-entrypoint-initdb.d
+
+    java:
+        image: openjdk:11
+        container_name: mapbot_java
+        volumes:
+            - java_storage:/usr/local/openjdk-11/
+
+    bot:
+        image: python:3.6
+        container_name: mapbot_bot
+        environment:
+            - DOCKER=true
```

Comment:
I'm suspecting this should be `True`, making the change locally did not resolve the issue however.

Reply:
The changes and discussion from the very next review deal with the same problem and I say let's continue the thread there.